### ISSUE TITLE
Upgrade via built-in updater, 3rd party apps

### ISSUE THREAD

**Issue by benediktfuss** [nextcloud/documentation#1513]
Include warning that 3rd party apps have to be disabled before running the web-updater. Add reference to [this section of manual](https://docs.nextcloud.com/server/16/admin_manual/maintenance/upgrade.html#prerequisites).

**Comment by skjnldsv** [comment]
@rullzer @MorrisJobke is this relevant?

**Comment by MorrisJobke** [comment]
Not really. They will be disabled automatically if they are not compatible.

**Comment by MorrisJobke** [comment]
So the sentence needs to be removed.
